wrappers/node/test/helpers/mockdata.ts

```diff
@@ -62,7 +63,7 @@ export const ARIES_CONNECTION_REQUEST = {
         {
           id: 'did:example:123456789abcdefghi;indy',
           priority: 0,
-          recipientKeys: ['2RjtVytftf9Psbh3E8jqyq#1'],
```

Comment:
Currently, after converting from the new to legacy DDO format, resolving key by reference fails.